### DISCUSSION TITLE
Chroma DB doesn't work

### DISCUSSION THREAD

**Discussion by devarthurguilherme** [langflow-ai/langflow#3566] [Q&A]
![chromaddd](https://github.com/user-attachments/assets/f9ee48ef-da37-4693-ab3f-10e5dc6783a7)
![chromadb](https://github.com/user-attachments/assets/16b7638b-7cb9-48bc-82f6-597271d9b29f)

**Comment by gauravdhiman** [comment]
Looks like you are using the cloud version and that has some quota limits.
If you are still developing your app and not in prod (non-scaled version), you can use docker to host it locally. Its a smooth out of box solution.

![image](https://github.com/user-attachments/assets/3ccf78af-7de2-42e5-b9dd-9bc56f22ceea)
